python/tvm/contrib/cutlass/conv2d_operation.py

```diff
@@ -0,0 +1,240 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=invalid-name, unused-wildcard-import, wildcard-import
+"""Generator for CUTLASS Conv2D kernels."""
+from .library import *
+
+
+class Conv2dOperation:
```

Comment:
It might be better to have a base classes (Operation/Emit/Profiler) for both Conv2d and Gemm to enforce consistent APIs.

Reply:
Operation classes are only used by the corresponding Emit class, and there is not much to share between EmitGemm and EmitConv2d. I prefer to keep this way rather than introducing unnecessary abstractions.